pyproject.toml

```diff
@@ -9,7 +9,7 @@ name = "capella-dockerimages-tests"
 requires-python = ">=3.9, <3.12"
 license = { text = "Apache-2.0" }
 authors = [{ name = "DB Netz AG" }]
-dependencies = ["docker", "pytest", "capellambse[decl]"]
+dependencies = ["docker", "pytest", "capellambse[decl]", "chardet"]
```

Comment:
Good catch. While I'm uncertain whether this was the root cause for some of our issues, it's still good to avoid using incompatible libraries. To address this, I've updated the docker-py version to be greater than or equal to 6.1.0, which resolves the compatibility problem.